sys/include/net/gnrc/ipv6/whitelist.h

```diff
@@ -37,8 +37,8 @@ extern "C" {
 /**
  * Maximum size of the whitelist.
  */
-#ifndef GNRC_IPV6_WHITELIST_SIZE
-#define GNRC_IPV6_WHITELIST_SIZE    (8)
+#ifndef CONFIG_GNRC_IPV6_WHITELIST_SIZE
+#define CONFIG_GNRC_IPV6_WHITELIST_SIZE    (8)
```

Comment:
Do we *have* to rename all config macros to be usable with kconfig?

Reply:
Yes, Kconfig generates the .config and .h files appending the `CONFIG_` symbol. Regardless of that, I think it's good to have configuration macros under a single namespace.

Reply:
Ok, were just wondering for future integrations.